app/js/templates/questions.hbs

```diff
@@ -8,32 +8,32 @@
     <div class="answers">
         {{#if option1}}
         <p>
-            <input type="radio" name="answer_{{id}}" value="1">{{option1}}
+            <input class="questionValue" type="radio" name="answer_{{id}}" value="1">{{option1}}
         </p>
         {{/if}}
         {{#if option2}}
         <p>
-            <input type="radio" name="answer_{{id}}" value="2">{{option2}}
+            <input class="questionValue" type="radio" name="answer_{{id}}" value="2">{{option2}}
         </p>
         {{/if}}
         {{#if option3}}
         <p>
-            <input type="radio" name="answer_{{id}}" value="3">{{option3}}
+            <input class="questionValue" type="radio" name="answer_{{id}}" value="3">{{option3}}
         </p>
         {{/if}}
         {{#if option4}}
         <p>
-            <input type="radio" name="answer_{{id}}" value="4">{{option4}}
+            <input class="questionValue" type="radio" name="answer_{{id}}" value="4">{{option4}}
         </p>
         {{/if}}
         {{#if option5}}
         <p>
-            <input type="radio" name="answer_{{id}}" value="5">{{option5}}
+            <input class="questionValue" type="radio" name="answer_{{id}}" value="5">{{option5}}
         </p>
         {{/if}}
     </div>
     <div class="textBox">
         <label>Question notes:</label>
-        <textarea name="answer_{{id}}" rows="7" cols="50"></textarea>
+        <textarea id="notes" name="answer_{{id}}" rows="7" cols="50"></textarea>
```

Comment:
you cant do this. now there are lots of elements with the id of `notes` on the page, which is invalid HTML

Reply:
you will have to use the class to select the correct notes textarea instead